egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -358,6 +358,8 @@ def train(args, run_opts):
                                right_context_final >= 0 else -1)
 
     default_egs_dir = '{0}/egs'.format(args.dir)
+    if (args.egs_dir is not None) and (args.cmvn_opts != "--norm-means=false --norm-vars=false"):
+        logger.warning("kaldi doesn't call get_egs.sh to cmvn the egs.")
```

Comment:
Can the warning say:
``` Warning: the --cmvn-opts option has no effect because we are not dumping egs```
You could put similar warnings for other options that only affect the egs dumping.  We recently had someone confused about the --frames-per-iter option.

Reply:
I have updated the pull request, may you take a look when you are available, thank you!